code/game/gamemodes/objective.dm

```diff
@@ -337,9 +337,9 @@ datum/objective/hijack
 		if(issilicon(owner.current))
 			return 0
 		var/area/shuttle = locate(/area/shuttle/escape/centcom)
-		var/list/protected_mobs = list(/mob/living/silicon/ai, /mob/living/silicon/pai)
 		for(var/mob/living/player in player_list)
-			if(player.type in protected_mobs)	continue
+			if(istype(player, /mob/living/silicon) || istype(player, /mob/living/simple_animal) || player.mind.special_role)

```

Comment:
A special role check is a bit too general here, it would mean that a traitor could get away with hijack with a deathsquad or ERT aboard, which is basically the opposite of hijacking the shuttle.